_src/assets/js/main.js

```diff
@@ -86,4 +86,31 @@ function applyPalette(e) {
 // En este caso, no se porque... no me funciona con 'click' y si con 'change'
 for (const c of colors) {
   c.addEventListener('change', applyPalette);
-}
\ No newline at end of file
+}
+
+// Añadir imagen
+
```

Comment:
Perfect job girls, very smart.
The olny thing is when I try to choose a image to upload, it ask me for a double confirmation and I don't know if it should do that or not.

Reply:
I reviewed it and it works fantastic! You can merge it

Reply:
Thanks both of you :)
Let's see if it works for @babelarr so we can definitely merge it.

Reply:
It's perfect!